Orange/widgets/data/owrank.py

```diff
@@ -71,8 +78,7 @@ def __new__(cls, name, shortname, score,
 
 class OWRank(widget.OWWidget):
     name = "Rank"
-    description = "Rank data features by their correlation to " \
-                  "the class variable."
+    description = "Ranks and filters data features by their relevance."
```

Comment:
Imperative is used in other descriptions and sounds better imho:
"Rank and filter data...